config/boot.rb

```diff
@@ -33,6 +33,11 @@
 puts "Loading config from #{env_file}"
 require env_file
 
+# Configure dor-services-client to use the dor-services URL
```

Comment:
do we need the comment?  the code reads cleanly

Reply:
Peter thought this was helpful. 🤷‍♂️  https://github.com/sul-dlss/common-accessioning/commit/c474925fb119643c3df828bc17fa4ff1578d49c4#diff-dfab7ed9f3a01a0f58ab1de265a6ae91R36